src/pymatgen/io/vasp/inputs.py

```diff
@@ -659,7 +659,7 @@ def from_dict(cls, dct: dict) -> Self:
     def set_temperature(self, temperature: float) -> None:
         """
         Initialize the velocities based on Maxwell-Boltzmann distribution.
-        Removes linear, but not angular drift (same as VASP).
+        Remove linear, but not angular drift (same as VASP).
```

Comment:
i would keep non-imperative "Removes"

Reply:
I think I changed this to keep consistency with the former sentence:
> Initialize the velocities based on Maxwell-Boltzmann distribution.

Reply:
Okay I misunderstand this here. Would revert this